calendar_reboot.js

```diff
@@ -354,9 +382,12 @@ $(document).ready(function() {
     }
     
     //load the current_active_calendar
+    //to check, check if active calendar is in localStorage
     var activeCalendarState = loadFromLocalStorage(calendarUniqueId[current_active_calendar]);
     
     if (activeCalendarState !== null) {
+        //create calendar object
+        calendar = new Calendar("", 0, "");
```

Comment:
I have to refactor the calendar object code, and then I will be able to modify these lines.